src/fdc3/dotnet/DesktopAgent/src/DesktopAgent/DesktopAgent.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="7.0.0" />
```

Comment:
We should consolidate package versions, but as a rule of thumb, use the 6.X version of these libraries when targeting .NET 6.